packages/tupaia-web/src/features/Map/MapLegend/DesktopMapLegend.tsx

```diff
@@ -13,19 +13,24 @@ const Wrapper = styled.div`
   flex-direction: row;
   align-items: flex-end;
   justify-content: center;
-  position: absolute;
-  background-color: grey;
-  width: 300px;
-  height: 50px;
-  bottom: 1em;
-  left: 50%;
-  transform: translateX(-50%);
-  border-radius: 5px;
+  width: 100%;
+  padding: 1rem;
   @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
     display: none;
   }
 `;
 
+const Legend = styled.div`
+  height: 50px;
```

Comment:
probably should be rems, right?

Reply:
It should, but this is just a placeholder so I don't think it's important at this stage

Reply:
oh true. I just see px and I jump 😆